doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_suit_fota_update.rst

```diff
@@ -0,0 +1,48 @@
+:orphan:
+
+.. _ug_nrf54h20_suit_fota_update:
+
+FOTA updates on nRF54H Series devices
+#####################################
+
+|fota_upgrades_def|
+You can also use FOTA updates to replace the application.
+
+FOTA over Bluetooth Low Energy
+******************************
+
+FOTA updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth.
+The application acts as a GATT server and allows the connected Bluetooth Central device to perform a firmware update.
+To use FOTA over Bluetooth LE, samples must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`).
+
+Testing steps
+=============
+
+To perform a FOTA update, complete the following steps:
+
+.. ug_nrf54h20_suit_fota_updates_over_ble_nrfcdm_common_dfu_steps_start
+
+1. Generate the SUIT envelope by building your application with the FOTA support over Bluetooth Low Energy.
+   You can find the generated :file:`root.suit` envelope in the :file:`<build_dir>/DFU` directory.
+   Alternatively, you can use the generated :file:`dfu_suit.zip` package in the :file:`<build_dir>/zephyr` directory.
+#. Download the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package to your device.
+
+   .. note::
+      `nRF Connect for Desktop`_ does not currently support the FOTA process.
+
+#. Use the `nRF Connect Device Manager`_ mobile app to update your device with the new firmware.
+
+   a. Ensure that you can access the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package from your phone or tablet.
+   #. In the mobile app, scan and select the device to update.
+   #. Switch to the :guilabel:`Image` tab.
+   #. In the **Firmware Upgrade** section, tap the :guilabel:`SELECT FILE` button and select the :file:`root.suit` envelope or the :file:`dfu_suit.zip` package.
+   #. Tap the :guilabel:`START` button.
+   #. Wait for the DFU to finish and then verify that the application works properly.
+
+.. note::
+   Support for SUIT updates is available starting from the following versions of the `nRF Connect Device Manager`_ mobile app:
+
+   * Version ``2.0`` on Android.
+   * Version ``1.7`` on iOS.
```

Comment:
Due to the recent modification in FW confirmation, the current set of version is:
```
* `nRF Device Manager mobile app for Android`_
  (The minimum required version is v2.2.1.)

* `nRF Device Manager mobile app for iOS`_
  (The minimum required version is v1.8.0.)
  ```

Reply:
fixed